lang/java/reef-tang/tang/src/test/java/org/apache/reef/tang/test/RoundTripTest.java

```diff
@@ -48,7 +49,8 @@ public void testRoundTrip() throws Exception {
   public void testRoundTripWithClassHierarchy() throws Exception {
     // TODO: use 'getConfiguration' instead of 'getConfigurationWithoutList' after #192 is fixed
     final Configuration confBefore = ObjectTreeTest.getConfigurationWithoutList();
-    final ClassHierarchy c = new ProtocolBufferClassHierarchy(ProtocolBufferClassHierarchy.serialize(confBefore.getClassHierarchy()));
+    final ClassHierarchySerializer chSerializer = new AvroClassHierarchySerializer();
+    final ClassHierarchy c = chSerializer.fromString(chSerializer.toString(confBefore.getClassHierarchy()));
```

Comment:
You're right. But I am not quite sure it is desirable to expose `fromAvro()` and `toAvro()` here, which should be the private methods I think. What do you think of removing this `ClassHierarchy` round trip since we have a separate test in the `TestClassHierarchyRoundTrip`?